subiquity/models/subiquity.py

```diff
@@ -176,11 +176,13 @@ def __init__(self, root, hub, install_model_names, postinstall_model_names):
             self.target = root
             self.chroot_prefix = []
 
+        log.debug("root is %s", root)
```

Comment:
debug stuff, is this meant for inclusion in the PR?

Reply:
Nope! thanks for noticing!